op-guide/migration-overview.md

```diff
@@ -1,58 +1,50 @@
 ---
 title: Migration Overview
-summary: Learn how to migrate data from MySQL to TiDB.
+summary: Learn how to migrate data into TiDB
 category: operations
 ---
 
 # Migration Overview
 
-## Overview
+This document describes scenarios for migrating data into TiDB from either MySQL or data source stored in CSV format.
 
-This document describes how to migrate data from MySQL to TiDB in detail.
+## Tools overview
 
-See the following for the assumed MySQL and TiDB server information:
+Migrations will often make use of the following tools. For a brief overview of their usage:
 
-|Name|Address|Port|User|Password|
-|----|-------|----|----|--------|
-|MySQL|127.0.0.1|3306|root|* |
-|TiDB|127.0.0.1|4000|root|* |
+- [`mydumper`](../tools/mydumper.md) exports data from MySQL. It is recommended over using mysqldump.
+- [`loader`](../tools/loader.md) imports data in mydumper format into TiDB
+- [`syncer`](../tools/syncer.md) acts like a MySQL replication slave and pushes data from MySQL into TiDB.
+- [DM](../tools/dm/overview.md) (Data Migration) integrates the functions of mydumper, Loader and syncer to support the export and import of full-size data, as well as incremental synchronization of MySQL Binlog data, and supports data synchronization of a more complete pooled table scenario.
+- [TiDB-Lightning](../tools/lightning/overview-architecture.md) imports data to TiDB in an optimized way.  For example, a 1TiB backup could take 10+ hours to import with loader, may be complete in TiDB-Lightning in just 3 hours.
```

Comment:
```suggestion
- [TiDB-Lightning](../tools/lightning/overview-architecture.md) imports data to TiDB in an optimized way. For example, a 1TiB backup could take 10+ hours to import with loader, while it may be completed in TiDB-Lightning in just 3 hours.
```

Reply:
Sorry, I edited in between your suggestion.  I've changed to your version now though.  Thanks!